atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -236,7 +236,7 @@ def _pull_and_tag_image(self, image, build_json, nonce):
                 # need to retry with that.
                 if first_library_exc:
```

Comment:
Maybe
```python
if first_library_exc is not None:
```
may make pylint happy

Reply:
If it won't make pylint happy, definitely it will make me happy to avoid a condition when exception implements `__bool__` returning `False` :D

Reply:
I even tried
```python
if isinstance(first_library_exc, Exception):
```
Looks like pylint is not _that_ smart :D